api/src/main/java/jakarta/json/spi/JsonProvider.java

```diff
@@ -57,15 +79,27 @@ protected JsonProvider() {
     }
 
     /**
-     * Creates a JSON provider object. The provider is loaded using the
-     * {@link ServiceLoader#load(Class)} method. If there are no available
-     * service providers, this method returns the default service provider.
+     * Creates a JSON provider object.
+     *
+     * Implementation discovery consists of following steps:
+     *
+     * 1. If the system property {@link #JSONP_PROVIDER_FACTORY} exists,
```

Comment:
I'd probably use `@value` instead of `@link` here, but no strong opinion on that